README.md

```diff
@@ -15,32 +15,32 @@ This will output a file named "poketcg.gbc".
 ## See also
 
 - [**Symbols**][symbols]
-- **Discord:** [pret][discord]
-- **IRC:** [libera#pret][irc]
-
-Other disassembly projects:
-
-- [**Pokémon TCG 2**][poketcg2]
-- [**Pokémon Red/Blue**][pokered]
-- [**Pokémon Yellow**][pokeyellow]
-- [**Pokémon Gold/Silver**][pokegold]
-- [**Pokémon Crystal**][pokecrystal]
-- [**Pokémon Pinball**][pokepinball]
-- [**Pokémon Ruby**][pokeruby]
-- [**Pokémon FireRed**][pokefirered]
-- [**Pokémon Emerald**][pokeemerald]
-
-[poketcg2]: https://github.com/pret/poketcg2
-[pokered]: https://github.com/pret/pokered
-[pokeyellow]: https://github.com/pret/pokeyellow
-[pokegold]: https://github.com/pret/pokegold
-[pokecrystal]: https://github.com/pret/pokecrystal
-[pokepinball]: https://github.com/pret/pokepinball
-[pokeruby]: https://github.com/pret/pokeruby
-[pokefirered]: https://github.com/pret/pokefirered
-[pokeemerald]: https://github.com/pret/pokeemerald
+- 
+Other disassembly and/or decompilation projects:
+* [**Pokémon Red and Blue**](https://github.com/pret/pokered)
+* [**Pokémon Gold and Silver (Space World '97 demo)**](https://github.com/pret/pokegold-spaceworld)
+* [**Pokémon Yellow**](https://github.com/pret/pokeyellow)
+* [**Pokémon Pinball**](https://github.com/pret/pokepinball)
+* [**Pokémon Stadium**](https://github.com/pret/pokestadium)
+* [**Pokémon Gold and Silver**](https://github.com/pret/pokegold)
+* [**Pokémon Crystal**](https://github.com/pret/pokecrystal)
+* [**Pokémon Ruby and Sapphire**](https://github.com/pret/pokeruby)
+* [**Pokémon Pinball: Ruby & Sapphire**](https://github.com/pret/pokepinballrs)
+* [**Pokémon FireRed and LeafGreen**](https://github.com/pret/pokefirered)
+* [**Pokémon Emerald**](https://github.com/pret/pokeemerald)
+* [**Pokémon Mystery Dungeon: Red Rescue Team**](https://github.com/pret/pmd-red)
+* [**Pokémon Diamond and Pearl**](https://github.com/pret/pokediamond)
+* [**Pokémon Platinum**](https://github.com/pret/pokeplatinum) 
+* [**Pokémon HeartGold and SoulSilver**](https://github.com/pret/pokeheartgold)
+* [**Pokémon Mystery Dungeon: Explorers of Sky**](https://github.com/pret/pmd-sky)
```

Comment:
At a certain point, it becomes silly to list every side game, and it would be just as effective and more maintainable to just link to the pret org and say "find other projects here".

Reply:
There's discussion about moving this list to a central location, which should serve that purpose.

Reply:
That phrasing is confusing to me. There already is a central location for the list. It's https://github.com/pret

Reply:
The list isn't ordered by generation, so it's not as user-friendly. I made a PR to [pret.github.io](https://github.com/pret/pret.github.io/pull/1) to serve as a more organized list to link to, though I don't think it's gained much traction.

Reply:
I didn't even know a github io page had been created. I don't feel like it is necessary.

Reply:
What do you want to do with the disassembly list then?
- Merge this PR to update the existing list, following pret's current readme convention.
- Open a discussion to change the current convention for all pret repos.
- Break convention with this repo only, and link to https://github.com/pret
- Leave this PR open, take no other action, and leave the readme's list in its current out-of-date state.

Reply:
Well I'm fine with this being merged for consistency. But the stylization of PRET instead of pret is already merged into several of the repos, and the lists in all of those repos are already out-of-date because they don't include poketcg2.

Reply:
The "PRET" stylization was already in a lot of the repos prior to my wave of PRs, so I had followed that originally before it was pointed out to use "pret". I was not aware of TCG 2 when making my PRs, since no repo other than this one mentioned it. You are welcome to make PRs to all the repos to fix them.